src/main/resources/application.yml

```diff
@@ -12,12 +12,6 @@ spring:
     ansi:
       enabled: ALWAYS
 
-  datasource:
-    driver-class-name: org.postgresql.Driver
-    url: ${SPRING_DATASOURCE_URL}
-    username: ${SPRING_DATASOURCE_USERNAME}
-    password: ${SPRING_DATASOURCE_PASSWORD}
-
```

Comment:
this must be a bad idea
please keep it in the `application.yaml` 
coz it's used across all envs and profiles. 
only test profile overrides config values with H2.

Reply:
I will solve this problem after refactoring Flyway